python/ptest/cases/dcdc_checkout_case.py

```diff
@@ -0,0 +1,100 @@
+from .base import SingleSatOnlyCase
+
+class DCDCCheckoutCase(SingleSatOnlyCase):
+
+    def setup_case_singlesat(self, simulation):
+        simulation.flight_controller.write_state(
+            "pan.state", 9)  # Manual state
+        self.run_case_singlesat(simulation)
+        print("DCDC cases finished.")
+
+    def run_case_singlesat(self, simulation):
+        simulation.cycle_no = simulation.flight_controller.read_state("pan.cycle_no")
+
+        adcs_cmd = simulation.flight_controller.read_state("dcdc.ADCSMotor_cmd")
+        sph_cmd = simulation.flight_controller.read_state("dcdc.SpikeDock_cmd")
+        disable_cmd = simulation.flight_controller.read_state("dcdc.disable_cmd")
+        reset_cmd = simulation.flight_controller.read_state("dcdc.reset_cmd")
+        adcs_pin = simulation.flight_controller.read_state("dcdc.ADCSMotor")
+        sph_pin = simulation.flight_controller.read_state("dcdc.SpikeDock")
+        if adcs_cmd==sph_pin and sph_cmd==sph_pin and disable_cmd=="false" and reset_cmd=="false":
+            print("Control task initialized correctly")
+
+        # Try both DCDCs on. Turn on all systems (ADCS motors, prop valves, docking motor). 
+
+        print ("Test case 1: ")
```

Comment:
I think it's a good idea to be more descriptive with the test case when printing to the console. No need for the comment on line 23: just put it directly into the print statement! That way you get the best of both worlds: inline documentation of the code, and documentation of the test as it prints to the screen.